docs/language/overview.md

```diff
@@ -457,13 +457,103 @@ scope but can override identifiers defined in ancestor scopes.
 
 `func` statements may appear intermixed with `const` and `type` statements.
 
-## 5. Type Statements
+## 5. Operator Statements
+
+User-defined operators may be created with the syntax
+
+```
+op <id> ( [<param> [, <param> ...]] ) : ( 
+  <statements>
+  <sequence>
+)
+```
+where `<id>` and `<param>` are identifiers, `<statements>` is a list of
+statements, and `<sequence>` is a chain of operators (e.g., `operator | ...`).
+
+A user-defined operator can then be called with using the familiar call syntax
+```
+<id> ( [<expr> [, <expr> ...]] )
+```
+where `<id>` is the identifier of the user-defined operator and `<expr>` is a list
+of [expressions](#8-expressions) matching the number of `<param>`s defined in
+the operator's signature.
+
+### 5.1 Scope Context
```

Comment:
Ugh, I hate the titles of these sections. I believe these concepts are important for the user to understand but am unsure how to explain them in simple, user facing language. "Inner" and "Outer" scope isn't really right since really its the same scope. The word "Context" is confusing but saying "the `this` value" everywhere also seems weird. I will think about this but am also open to suggestion.